### PR TITLE
Reorganize spec for irregular pluralization

### PR DESCRIPTION
**Pull request by abinoam**
This is just a "cherry pick" of parts of #10 
I have preserved the authorship of the commit.

### CODE REVIEW

File: spec/unit/dry/inflector/pluralize_spec.rb [hunk 1]
```diff
@@ -8,6 +8,12 @@
       end
     end
 
+    Fixtures::Pluralize.irregular.each do |singular, plural|
+      it "#{singular} => #{plural} is idempotent" do
+        expect(subject.pluralize(i(singular))).to eq(plural)
```
Comment (jodosha) [inline spec/unit/dry/inflector/pluralize_spec.rb]: @abinoam I don't understand the purpose of this line. Isn't it identical to the existing tests for regular pluralizations?

**Comment by abinoam** [comment]
Hi @jodosha, I think it's useful to separate irregular from regular pluralization fixtures.
Then I just chery picked this part of your PR.
But, I forgot to update spec description, as you have noted.
I've just fixed it.

**Comment by solnic** [comment]
@abinoam please use this repo for branches/PRs instead of your fork, this way it's much easier to pull stuff locally (if needed), also, the less forks we have on github, the better :)

**Comment by abinoam** [comment]
👍 @solnic 
Deleting my fork.

**Pull request merged**